content/references/gamification/privileges.md

```diff
@@ -0,0 +1,23 @@
+---
+kind: reference
+sidebar: ranks-honor
+prev: /references/gamification/honor/
+---
+
+# Privileges
+
+| Privilege                                 | Required Honor | Description                                                  |
+| ----------------------------------------- | -------------: | ------------------------------------------------------------ |
+| Vote Kata                                 |             25 | Vote on how satisfied you were with a kata                   |
+| Mark comment as spoiler                   |             50 | Mark another's comment as a spoiler                          |
+| Estimate ranking of own beta kata         |             75 | Estimate on what rank you think your beta kata should be     |
+| Assess Rank                               |            100 | Vote on what rank you think a beta kata should be            |
+| Create Kata                               |            300 | Contribute your own kata to the community                    |
+| Unmark comment as spoiler                 |            500 | Unmark another's comment as being a spoiler                  |
+| Vote 2x power                             |          1,000 | Your vote counts 2x towards getting a kata out of beta       |
+| Vote 3x power                             |          2,000 | Your vote counts 3x towards getting a kata out of beta       |
+| Vote 4x power                             |          3,000 | Your vote counts 4x towards getting a kata out of beta       |
+| Coauthor Kata **†** / Approve Translation |          4,000 | Ability to edit other author's kata and approve translations |
```

Comment:
`†` is on a separate page now.

Reply:
I duplicate the note here or I link to the other page?

Reply:
Move the note here because it doesn't make sense on that page by itself.

Reply:
```suggestion
| Coauthor Kata / Approve Translation |          4,000 | Ability to edit other author's kata and approve translations |
```